background/get-userscripts.js

```diff
@@ -17,61 +17,104 @@ function getL10NURLs() {
   return urls;
 }
 
+scratchAddons.localEvents.addEventListener("addonEnabled", ({ detail }) => {
+  const { addonId, manifest } = detail;
+  chrome.tabs.query({}, (tabs) =>
+    tabs.forEach((tab) => {
+      if (tab.url || (!tab.url && typeof browser !== "undefined")) {
+        chrome.tabs.sendMessage(tab.id, "getInitialUrl", { frameId: 0 }, async (res) => {
```

Comment:
I'm not sure Firefox likes async callbacks, I had trouble with them once

Reply:
lol then should I send a promise?

Reply:
No. Make it a sync function. If you need to use await, use `.then()` or an async IIFE inside the non-async function.